### PR TITLE
fix(HttpSourceAcl): Add optional configuration to require Content-Length

### PR DESCRIPTION
**Pull request by NicolasFruyAdeo**
Defaults to false, fail fast rather than skipping HTTP response body length validation.

### CODE REVIEW

**Comment by NicolasFruyAdeo** [comment]
Superseded by #2.

File: src/main/scala/io/conduktor/ksm/source/HttpSourceAcl.scala [hunk 1]
```diff
@@ -81,21 +123,25 @@ class HttpSourceAcl(parserRegistry: AclParserRegistry)
...
     authentication.map(authentication => request.header(authentication.authHeaderKey, authentication.authHeaderValue))
 
     // we use this header for the 304
-    lastModified.foreach(header => request.header("If-Modified-Since", header))
-    request.header("Content-Type", "text/plain") // only type supported for now
+    lastModified.foreach(header => request.header(IF_MODIFIED_SINCE, header))
+    request.header(CONTENT_TYPE, "text/plain") // only type supported for now
+
+    this.headers.foreach {case (name, value) => request.header(name, value)}
+
```
Comment (brunodomenici) [inline src/main/scala/io/conduktor/ksm/source/HttpSourceAcl.scala:139]: Maybe we should put this before line 132 to make sure that if there is an authentication header in the conf it won't overwrite authentication header

**Review by brunodomenici — APPROVED** [review]
Great. Just a minor comment about setting order of http header

File: src/main/scala/io/conduktor/ksm/source/HttpSourceAcl.scala (hunk 1 above)
Comment (NicolasFruyAdeo) [reply src/main/scala/io/conduktor/ksm/source/HttpSourceAcl.scala:139]: Sure, I was considering those extra headers as overrides, but happy to move this wherever you want.

**Pull request closed without merging**